javascript/NocaptchaField.js

```diff
@@ -3,12 +3,11 @@ var _noCaptchaFields=_noCaptchaFields || [];
 function noCaptchaFieldRender() {
     var submitListener=function(e) {
         e.preventDefault();
-        
-        grecaptcha.execute();
+        let widgetID = e.target.querySelectorAll('.g-recaptcha')[0].getAttribute('data-widgetid');
```

Comment:
I know it's old school but let's use `var` instead of let, maintains better compatibility just in case :)

Reply:
Done! Sorry for the slow response been busy as

Reply:
All good :)